test/test_unit.py

```diff
@@ -10,66 +10,74 @@
 from steamship.plugin.inputs.raw_block_and_tag_plugin_input_with_preallocated_blocks import (
     RawBlockAndTagPluginInputWithPreallocatedBlocks,
 )
-from steamship.plugin.outputs.plugin_output import UsageReport, OperationUnit
+from steamship.plugin.outputs.plugin_output import UsageReport, OperationUnit, OperationType
 from steamship.plugin.request import PluginRequest
 
-from src.api import GPT4Plugin
+from src.api import LiteLLMPlugin
 
+FUNCTION_MODEL_PARAMS = ["", "gpt-4-32k"]
```

Comment:
What does the `gpt-4-32k` do here?

Reply:
This was the original parametrization of the models for the tests, so I'm persisting them this way.